packages/MarkdownEditor-Tests.package/MarkdownCodeBlockTest.class/instance/testFenceLength3To6.st

```diff
@@ -0,0 +1,12 @@
+tests
+testFenceLength3To6
+
+	(Dictionary newFrom: {'```'->3. '```smalltalk'->3.
```

Comment:
I would like to leave a newline before the dictionary starts, that way we would have all entries nicely in one block, visually